.github/workflows/pre-integration.yml

```diff
@@ -16,24 +16,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        java: [7, 8, 11, 17, 19-ea]
-        os: [ubuntu-20.04, windows-2022, macos-11]
+        java: [8, 11, 17, 19, 20-ea]
```

Comment:
Huh, interesting, unlucky 13 releases old J7 is not supported by 20 :)
Since the problem seems to be with compilation, this is an interesting blog post this project (probably not in scope of this PR, but something to muse on): https://jakewharton.com/build-on-latest-java-test-through-lowest-java/

Reply:
Honestly, it might be just a good time to drop JDK 7 support: #86.

Reply:
Nice PR! The blog post still applies though for the future ;)